packages/amplify-graphql-schema-generator/package.json

```diff
@@ -19,7 +19,7 @@
     "generator"
   ],
   "scripts": {
-    "build": "tsc",
+    "build": "tsc && cd vpc-db-lambda && rm -rf node_modules && npm install && tsc && cp -r node_modules lib && cd lib && bestzip --force node ../../lib/rds-schema-inspector.zip ./*",
```

Comment:
Not critical for this PR, but `rm -rf` and `cp -r` will fail on windows build machines.